src/tv2-common/inewsConversion/converters/ParseBody.ts

```diff
@@ -675,7 +675,7 @@ export function getSourceDefinition(typeStr: string): SourceDefinition | undefin
 		}
 	} else if (EVS_RED_TEXT.test(typeStr)) {
 		const strippedToken = typeStr.match(EVS_RED_TEXT)
-		const id = `EVS ${strippedToken![1].toUpperCase()}`
+		const id = `EVS ${strippedToken![1].toUpperCase()} 1/2`
```

Comment:
This will find the correct configuration, but will give the incorrect piece name. When `EVS 1` is written in iNews, the corresponding piece name should be `EVS 1` as well. It is only action pieces that should show the audio track suffix.

Reply:
Good catch.
I have updated it so the name does not get the audio track (the `1/2`).